ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -198,6 +198,13 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				Description: "Labels for default worker pool",
 			},
 
+			"operating_system": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Operating system of the default worker pool. Options are REDHAT_7_64 or REDHAT_8_64.",
```

Comment:
The options can change, so I don't think it's valid for us to list them here. We can link to the documentation or API if it lists them.

```suggestion
				Description: "The operating system of the workers in the default worker pool.",
```